processor/core/src/main/java/com/redhat/service/bridge/processor/actions/kafkatopic/KafkaTopicActionBean.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.service.bridge.processor.actions.kafkatopic;
+
+import com.redhat.service.bridge.processor.actions.ActionBean;
+
+public interface KafkaTopicActionBean extends ActionBean {
```

Comment:
@kostola I'm not sure I really see the value in `KafkaTopicActionBean` and `KafakTopicAction`. IMHO, we can simplify and have `KafkaTopicAction` implement `ActionBean`.

Same for all the other Actions.

Reply:
I had the same idea but I wasn't sure that it could cause confusion so I went with the split.

@r00ta @lucamolteni @danielezonca what is your opinion on this? Let's go with the majority :smile:

Reply:
@kostola What confusion were you trying to avoid with the split?

Reply:
Resolving this because this part has been refactored